.github/workflows/ci.yml

```diff
@@ -92,23 +93,20 @@ jobs:
           cache: "pip"
           cache-dependency-path: "setup.py"
 
-      - name: Upgrade pip and virtualenv to latest
-        run: pip install --upgrade pip virtualenv
```

Comment:
Why don't we want to upgrade pip / virtualenv to latest anymore?

Reply:
We use pip from python 3.10 and use venv instead of virtualenv that's bundled with python as well.